src/app/components/navbar/navbar.component.html

```diff
@@ -0,0 +1,51 @@
+<section class="main-container">
+  <div class="navbar">
+    <div class="first-col">
+      <div class="pic-container H">
+        <img src="./assets/images/svg/menu.svg" class="menu-icon" alt="menu">
+      </div>
+      <div class="first-col" *ngIf="!isArchiveRoute()">
+        <div class="keep-img">
+          <img src="../../../assets/images/keep.png" width="40">
```

Comment:
why we need to add ../../../ to access keep.png whereas we can easily get menu.svg without it?